exporter/src/app/http/export_artboards.cljs

```diff
@@ -0,0 +1,64 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) UXBOX Labs SL
+
+(ns app.http.export-artboards
+  (:require
+   ["path" :as path]
+   [app.common.exceptions :as exc :include-macros true]
+   [app.common.spec :as us]
+   [app.renderer.pdf :as rp]
+   [app.util.shell :as sh]
+   [cljs.spec.alpha :as s]
+   [cuerdas.core :as str]
+   [promesa.core :as p]))
+
+(s/def ::name ::us/string)
+(s/def ::file-id ::us/uuid)
+(s/def ::page-id ::us/uuid)
+(s/def ::frame-id ::us/uuid)
+(s/def ::frame-ids (s/coll-of ::frame-id :kind vector?))
+
+(s/def ::handler-params
+  (s/keys :req-un [::file-id ::page-id ::frame-ids]))
+
+(defn export-artboards-handler
+  [{:keys [params cookies] :as request}]
+  (let [{:keys [name file-id page-id frame-ids]} (us/conform ::handler-params params)
+        token  (.get ^js cookies "auth-token")
+
+        export-frame (fn [tdpath frame-id]
```

Comment:
I think this functions can be defined as top level functions.

Reply:
done